spec/facets/ColumnLineageDatasetFacet.json

```diff
@@ -32,19 +32,38 @@
                         "field": {
                           "type": "string",
                           "description": "The input field"
+                        },
+                        "transformations": {
+                          "type": "array",
+                          "items": {
+                            "type": "object",
+                            "properties": {
+                              "type": {
+                                "description": "The type of the transformation.",
+                                "type": "string",
+                                "enum": ["IDENTITY", "TRANSFORMED", "INDIRECT"]
```

Comment:
After previous experiences with `enum` fields with spec we prefer to use just string, and represent enum values in a textual way - since having raw enum prevents us from being forward compatible on a backend.

Reply:
done